.pre-commit-config.yaml

```diff
@@ -25,6 +25,10 @@ repos:
     hooks:
     -   id: ruff
         args: [--exit-non-zero-on-fix]
+    -   id: ruff
+        # TODO: remove this when ANN204 is enabled (ANN001 might have autofixes in the future)
```

Comment:
shall we add an alias to this, so people don't see `ruff` twice and risk confusion?
something like
```
- id: ruff
  name: ruff-selected-autofixes
  alias: ruff-selected-autofixes
```
?

Reply:
Okay - could then also re-use it for other autofixes in the future.